roles/pcp/README.rst

```diff
@@ -53,6 +53,10 @@ connect to hosts that it monitors (in seconds)::
 
     pmcd_connect_timeout: 1
 
+To tell a ``pcp_manager`` host to retain full-resolution archives for a year::
```

Comment:
can we have a reference to the doc that describes legal forms of this param (the example makes it clear that "Ndays" is valid, but can I say hours, or weeks, or...)